relay-server/src/utils/unreal.rs

```diff
@@ -47,9 +48,10 @@ fn get_event_item(data: &[u8]) -> Result<Option<Item>, Unreal4Error> {
 pub fn expand_unreal_envelope(
     unreal_item: Item,
     envelope: &mut Envelope,
+    config: &Config,
 ) -> Result<(), Unreal4Error> {
     let payload = unreal_item.payload();
-    let crash = Unreal4Crash::parse(&payload)?;
+    let crash = Unreal4Crash::parse_with_limit(&payload, config.max_envelope_size())?;
```

Comment:
are we sure we want to reuse envelope-size here?

unreal crash report can be minidumps right?

Reply:
Correct. The envelope size limit is the maximum size any envelope can ever get, including minidumps, attachments, and event data.

Reply:
ah okay we have minidumps in envelopes already, right